readthedocs/projects/models.py

```diff
@@ -1334,8 +1334,9 @@ def show_advertising(self):
         """Whether this project is ad-free (don't access the database)."""
         return not self.ad_free
 
-    @property
-    def environment_variables(self):
+    def environment_variables(self, *, all=True):
```

Comment:
I think this should keep being a property. I understand it will be populated with what's returned by the API, so there is no need to re-filter by "only public" environment vars.

Reply:
But isn’t the idea that this is a drop-in replacement for the real model? where is it used?

Reply:
OK, done by leaving the default as “return all”, see above